CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to hypersdk
+
+Thank you for your interest in contributing to `hypersdk`! By contributing to hypersdk, you are helping to build the foundation for the next generation of blockchains and decentralized applications.
+
+## Getting Started
+
+### Prerequisites
+
+To contribute to `hypersdk`, you'll need:
+
+- [Go](https://golang.org/dl/) 1.19 or higher
+
+### Setting up your development environment
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/ava-labs/hypersdk.git
+cd hypersdk
+```
+
+2. Install the dependencies:
+
+```go
+go mod download
+```
+
+This will download and install all required dependencies for the project.
+
+3. Build the project:
+
+```go
+go build ./...
+```
+This will build the project and its dependencies.
+
+## Building and running tests
+
+To build and run tests for the hypersdk, simply run:
+
+```go
+go test -timeout 30s ./...
+```
+
+This will build and run all tests for the project.
+
+## Building and running the project
+
+> TODO: How to build and run the project
+
+## Running linters
+
+> TODO: How to lint and format 
```

Comment:
This is just:
```
./scripts/tests.lint.sh
```

Reply:
We also have a "fixer" that tries to help: 
```
./scripts/fix.lint.sh
```

Reply:
Added, I wasn't sure about how to test everything. Didn't notice the `scripts` folder, thank you!